src/Localization/en/GraphAPICompute.json

```diff
@@ -0,0 +1,58 @@
+  "GremlinV2": "GremlinV2",
+  "LearnAboutCompute": "Learn more about GraphAPI Compute.",
+  "DeprovisioningDetailsText": "Learn more about deprovisioning the GraphAPI Compute.",
+  "ComputePricing": "Learn more about GraphAPI Compute pricing.",
+  "SKUs": "SKUs",
+  "SKUsPlaceHolder": "Select SKUs",
+  "NumberOfInstances": "Number of instances",
+  "CosmosD4s": "Cosmos.D4s (General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory)",
+  "CosmosD8s": "Cosmos.D8s (General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory)",
+  "CosmosD16s": "Cosmos.D16s (General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory)",
+  "CosmosD32s": "Cosmos.D32s (General Purpose Cosmos Compute with 32 vCPUs, 128 GB Memory)",
+  "CreateMessage": "Graph GraphAPI Compute resource is being created.",
+  "CreateInitializeTitle": "Provisioning resource",
+  "CreateInitializeMessage": "GraphAPI Compute resource will be provisioned.",
+  "CreateSuccessTitle": "Resource provisioned",
+  "CreateSuccesseMessage": "GraphAPI Compute resource provisioned.",
+  "CreateFailureTitle": "Failed to provision resource",
+  "CreateFailureMessage": "GraphAPI Compute resource provisioning failed.",
+  "UpdateMessage": "GraphAPI Compute resource is being updated.",
+  "UpdateInitializeTitle": "Updating resource",
+  "UpdateInitializeMessage": "GraphAPI Compute resource will be updated.",
+  "UpdateSuccessTitle": "Resource updated",
+  "UpdateSuccesseMessage": "GraphAPI Compute resource updated.",
+  "UpdateFailureTitle": "Failed to update resource",
+  "UpdateFailureMessage": "GraphAPI Compute resource updation failed.",
+  "DeleteMessage": "GraphAPI Compute resource is being deleted.",
+  "DeleteInitializeTitle": "Deleting resource",
+  "DeleteInitializeMessage": "GraphAPI Compute resource will be deleted.",
+  "DeleteSuccessTitle": "Resource deleted",
+  "DeleteSuccesseMessage": "GraphAPI Compute resource deleted.",
+  "DeleteFailureTitle": "Failed to delete resource",
+  "DeleteFailureMessage": "GraphAPI Compute resource deletion failed.",
+  "CannotSave": "Cannot save the changes to the GraphAPI Compute resource at the moment.",
+  "GraphAccountEndpoint": "Graph Account Endpoint",
+  "NoValue": "",
+  "CosmosD4Details": "General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory",
+  "CosmosD8Details": "General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory",
+  "CosmosD16Details": "General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory",
+  "ApproximateCost": "Approximate Cost Per Hour",
+  "CostText": "Hourly cost of the GraphAPI Compute resource depends on the SKU selection, number of instances per region, and number of regions.",
+  "ConnectionString": "Connection String",
+  "ConnectionStringText": "To use the GraphAPI Compute, use the connection string shown in ",
+  "KeysBlade": "the keys blade.",
+  "MetricsString": "Metrics",
+  "MetricsText": "Monitor the CPU and memory usage for the GraphAPI Compute instances in ",
+  "MetricsBlade": "the metrics blade.",
```

Comment:
goes Uluru already have this integration?

Reply:
No not yet, but it is WIP. I would come back to change it once we have the billing integration.